packages/webapp/src/containers/MediaWithAuthentication/index.jsx

```diff
@@ -1,14 +1,17 @@
 import { useEffect, useState } from 'react';
 import { mediaEnum } from './constants';
 import { ReactComponent as Download } from '../../assets/images/farmMapFilter/Download.svg';
+import * as JSZip from 'jszip';
```

Comment:
It looks like this does not work with storybook ([story](http://localhost:6006/?path=/story/components-documenttile--variety)). Do we have a solution?